dev-tools/packaging/templates/linux/postinstall.sh.tmpl

```diff
@@ -47,5 +17,9 @@ fi
 echo "create symlink "$symlink" to "$new_agent_dir/elastic-agent""
 ln -s "$new_agent_dir/elastic-agent" "$symlink"
 
+# reload systemctl and then restart service
+echo "systemd enable/restart elastic-agent"
 systemctl daemon-reload 2> /dev/null
+systemctl enable elatic-agent 2> /dev/null || true
```

Comment:
I notice there's a typo in the service name here

Reply:
Thanks for the catch. I have updated this.